server/sonar-server/src/main/java/org/sonar/server/startup/RegisterMetrics.java

```diff
@@ -103,8 +103,9 @@ private void save(DbSession session, Iterable<Metric> metrics) {
 
     for (MetricDto nonUpdatedBase : basesByKey.values()) {
       if (!nonUpdatedBase.isUserManaged()) {
-        LOG.info("Disable metric {} [{}]", nonUpdatedBase.getShortName(), nonUpdatedBase.getKey());
-        dbClient.metricDao().disableCustomByKey(session, nonUpdatedBase.getKey());
+        if (dbClient.metricDao().disableCustomByKey(session, nonUpdatedBase.getKey())) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1066)